### PR TITLE
Updated Code of Conduct Section

### PR DESCRIPTION
**Pull request by kariljordan**
This section was revised per #897 to revise the section that speaks about the Code of Conduct. It includes mention of Incident Response Checklists for instructors and other community members.

### CODE REVIEW

**Comment by karenword** [comment]
Thank you for this PR @kariljordan ! We discussed this in trainer training today, and there was some concern about removing content that specifically educates instructors about actions they are empowered to take. I'm not sure this was the best content to begin with, but is there anything we can replace that text with that is consistent with our other current CoC documentation?

**Comment by kariljordan** [comment]
@karenword I added the language back, but kept "intent" out of the language because the CoC was revised such that we do not consider whether someone _intended_ to break the CoC.

File: _episodes/18-management.md [hunk 1]
```diff
@@ -87,17 +87,20 @@ and hope that they'll understand.
...
+
+You are encouraged to move from being a bystander to being a Code of Conduct first responder. If you see something inappropriate happening, speak up. If you don’t feel comfortable intervening, but feel someone should, please submit a report via the [Code of Conduct incident report form](https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl_6rkVtBIVWbb9YNOHQP_XaANDClmVNu0zs-w/viewform) to the Code of Conduct committee.
 
 The local workshop host is expected to help enforce the Code of Conduct and
 you can ask them to mediate an incident if you are uncomfortable or unable to do so yourself.
-If you choose to settle the issue yourself, you should notify the workshop host
-of the issue in case s/he feels additional steps should be taken.
+If you choose to settle the incident yourself, you should notify the workshop host
+of the incident, and inform the Code of Conduct committee by completing the [Incident Report Form(https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl_6rkVtBIVWbb9YNOHQP_XaANDClmVNu0zs-w/viewform).
```
Comment (mikemhenry) [inline _episodes/18-management.md]:
```suggestion
of the incident, and inform the Code of Conduct committee by completing the [Incident Report Form](https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl_6rkVtBIVWbb9YNOHQP_XaANDClmVNu0zs-w/viewform).
```

Comment (mikemhenry) [reply _episodes/18-management.md]: Just adding a dropped `]`

**Comment by karenword** [comment]
Yay thank you @kariljordan! I should have anticipated the possibility of an even more vigorous review in our second trainer-training meeting and waited to submit that first comment... but at least I can report that they really liked the update! :)  I will detail their additional questions and suggestions here to get them into the conversation. However, I think that some or most of these will need to be re-routed into issues as the scope of this PR is (appropriately) limited. We can do that once you've determined whether any of them can be easily addressed here. These are:

- concern about immediate resources for those uncomfortable responding directly. While filling out the form is adequate for situations that don't present an ongoing problem, it is not a useful alternative to taking action in more pressing situations. At times there may be no viable alternative to taking action of some kind, and they may need to work as a team to determine the best course of action instead.
- questions about a hierarchy of reporting. Does the process of notifying instructors, hosts, and Carpentries have an order? The role of the host was a particular point of confusion.
- questions about responsibility for determining what's reportable. For an example, if an instructor should feel free to report incidents in which they are unsure that the CoC has been violated because the role of the Committee is to make that determination, that could be stated explicitly.
- interest in a short summary of the checklist
- interest in a corresponding activity (definitely an issue for a future release)

Finally one thing that I have just noted in reviewing the linked documentation: it looks like apology is something that is specifically recommended _against_ in the checklist, so it might be best to remove that particular option here. As an alternative, I suggest adding "create or use a break to discuss the matter privately or among co-instructors" to help protect against instructors feeling trapped at the front of the room.

**Comment by kariljordan** [comment]
> Yay thank you @kariljordan! I should have anticipated the possibility of an even more vigorous review in our second trainer-training meeting and waited to submit that first comment... but at least I can report that they really liked the update! :) I will detail their additional questions and suggestions here to get them into the conversation. However, I think that some or most of these will need to be re-routed into issues as the scope of this PR is (appropriately) limited. We can do that once you've determined whether any of them can be easily addressed here. These are:
> 
> * concern about immediate resources for those uncomfortable responding directly. While filling out the form is adequate for situations that don't present an ongoing problem, it is not a useful alternative to taking action in more pressing situations. At times there may be no viable alternative to taking action of some kind, and they may need to work as a team to determine the best course of action instead.

I agree, but I do not think we have the capacity to offer immediate resources outside of recommending that they identify emergency response protocols for the location of the workshop/event (if in-person).

> * questions about a hierarchy of reporting. Does the process of notifying instructors, hosts, and Carpentries have an order? The role of the host was a particular point of confusion.

There is no order/hierarchy. If you'd like that to be made clear we can ask @karinlag to submit a PR with language. We want everyone (instructor, host, CoCc) to be made aware, unless there is a conflict of interest.

> * questions about responsibility for determining what's reportable. For an example, if an instructor should feel free to report incidents in which they are unsure that the CoC has been violated because the role of the Committee is to make that determination, that could be stated explicitly.

Yes, we can state that explicitly. Please feel free to submit a PR because I am not sure which section/document the trainers want that added to. (cc @karinlag)

> * interest in a short summary of the checklist

[Here](https://docs.carpentries.org/topic_folders/policies/incident-response.html) is the summary of the checklist (Heading: **Immediate Response**)

If the placement is an issue, feel free to submit a PR to move it, or we can ping @karinlag. 

> * interest in a corresponding activity (definitely an issue for a future release)
 
I am not sure what you mean by this.

> Finally one thing that I have just noted in reviewing the linked documentation: it looks like apology is something that is specifically recommended _against_ in the checklist, so it might be best to remove that particular option here. As an alternative, I suggest adding "create or use a break to discuss the matter privately or among co-instructors" to help protect against instructors feeling trapped at the front of the room.

I like the idea of removing the apology language here, and adding the sentence you came up with. That's a great idea. Go right ahead!

**Comment by karinlag** [comment]
A couple of comments: 

First, I am hesitant in how we phrase the language around notifying others. It should be done in coordination with the reporter. It can be difficult enough for someone to notify anybody at all, and if it becomes clear that many people ("many" can here be anything "more than one") will know, they might just not say anything.

Second, this issue intersects with a few other issues. First, we are currently developing a "Duty officer" role. The short version of that role is a person at a workshop (probably an instructor) that is the designated contact person for CoC issues. Second, once that is settled, we will update the language regarding how dealing with CoC issues are written about in the handbook. I think it might be a good idea to hold off on this one until we have figured how this is going to work.

**Comment by kariljordan** [comment]
That sounds good to me @karinlag. In terms of this issue, do you recommend leaving it open, or how would you like to proceed?

**Comment by karinlag** [comment]
@kariljordan I recommend that we leave this one open pending resolution of the Duty officer role, so that those things acan percolate through the other issues in a coherent manner :slightly_smiling_face:

**Comment by Denubis** [comment]
@karenword should your status:wait be updated?

**Pull request closed without merging**